autobot-backend/migrations/versions/20260906_088_llc_company_ceos.py

```
@@ -0,0 +1,141 @@
+Postgres only, like #15763's backfill and for the same reason: the statements
+use ``gen_random_uuid()``, which SQLite does not have, and the SQLite path
+exists so unit tests can create the schema. There are no pre-existing companies
+in an in-memory test database to provision for.
+
+The default holder is an **agent**, which is the owner's decision and matches
+the researched reference design. It is a default and not a definition: the
+column shape is the ``holder_type`` + ``holder_*_id`` keyspace, so a person can
+hold the position without a migration.
+"""
+
+from typing import Sequence, Union
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)